robot/impl/local_robot_test.go

```diff
@@ -259,30 +259,23 @@ func TestConfigRemote(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, len(statuses), test.ShouldEqual, 3)
 
-	armStatus := &armpb.Status{
-		EndPosition:    &commonpb.Pose{X: 500, Z: 300, OZ: 1},
-		JointPositions: &armpb.JointPositions{Values: []float64{0.0}},
-	}
 	convMap := &armpb.Status{}
 	decoder, err := mapstructure.NewDecoder(&mapstructure.DecoderConfig{TagName: "json", Result: &convMap})
 	test.That(t, err, test.ShouldBeNil)
 	err = decoder.Decode(statuses[0].Status)
 	test.That(t, err, test.ShouldBeNil)
```

Comment:
none of these actually test anything at this point, can we remove?

Reply:
Could you actually look at `TestStatus` and `TestStatusRemote` and let me know if I should remove those as well?